tests/test_jointcal_lsstSim.py

```diff
@@ -0,0 +1,94 @@
+# See COPYRIGHT file at the top of the source tree.
+
+from __future__ import division, absolute_import, print_function
+
+import matplotlib
+# Have to import matplotlib at the top level, because it can be imported by
+# something else further in the stack and then I can't reset the backend.
+matplotlib.use('Agg')
+
+import unittest
+import os
+
+from astropy import units as u
+
+import lsst.afw.geom
+import lsst.afw.coord
+import lsst.utils
+import lsst.pex.exceptions
+import jointcalTestBase
+
+try:
+    data_dir = lsst.utils.getPackageDir('validation_data_jointcal')
+    os.environ['ASTROMETRY_NET_DATA_DIR'] = os.path.join(data_dir, 'twinkles1_and_index')
+except lsst.pex.exceptions.NotFoundError:
+    data_dir = None
+
+# We don't want the absolute astrometry to become significantly worse
+# than the single-epoch astrometry (about 0.040").
+# This value was empirically determined from the first run of jointcal on
+# this data, and will likely vary from survey to survey.
+absolute_error = 42e-3*u.arcsecond
+# Set to True for a comparison plot and some diagnostic numbers.
+do_plot = False
+
+
+# for MemoryTestCase
+def setup_module(module):
+    lsst.utils.tests.init()
+
+
+class JointcalTestLSSTSim(jointcalTestBase.JointcalTestBase, lsst.utils.tests.TestCase):
+    def setUp(self):
+        jointcalTestBase.JointcalTestBase.setUp(self)
+        self.do_plot = do_plot
+        self.match_radius = 0.1*lsst.afw.geom.arcseconds
+
+        # position of the Twinkles run 1 catalog
+        center = lsst.afw.coord.IcrsCoord(53.00914*lsst.afw.geom.degrees, -27.43895*lsst.afw.geom.degrees)
+        radius = 3*lsst.afw.geom.degrees
+        self._prep_reference_loader(center, radius)
+
+        self.input_dir = os.path.join(data_dir, 'twinkles1')
+        self.all_visits = range(840, 850)
```

Comment:
Those are the visits in the validation data set? They'll always be hard coded in, and all_visits is documented in `jointcalTestBase.py`